### DISCUSSION TITLE
Input Voltage of USB-C Port?

### DISCUSSION THREAD

**Discussion by gtilford** [tmiw/ezDV#44] [Q&A] [answered]
What is the input voltage of the USB-C port on ezDV?  I want to power the unit off my station power and most adapters use USB-PD Decoy instead of negotiating the voltage required.  I am looking at something along the lines of this: https://www.amazon.com/AITRIP-Charging-Trigger-Detector-Terminal/dp/B098WPSMV9

**Comment by tmiw** [comment, marked as answer]
ezDV doesn't support Power Delivery so it only runs on 5V. There is circuity to detect the maximum power the USB-C power supply or PC will provide, which is used mainly to control how quickly the battery recharges (the onboard ESP32-S3 and TLV320 only use 100-150mA or so combined at most with Wi-Fi turned on).

That said, there is an eFuse on the board that will cap the voltage to 5-6V in case more than 5V accidentally gets fed into it but I wouldn't recommend trying to run off of it for long periods of time.

**Reply by gtilford** [reply]
That makes it even easier.  I'll power it the off the same MeanWell power supply as my RaspberryPi's.  73, Greg, KM5GT
